docs/site/Model.md

```diff
@@ -190,9 +188,188 @@ class Product extends Entity {
+      </li>
+      <li><code>false</code>: The model is an open model and accepts all properties, including ones not predefined in the model.
+        This mode is useful to store free-form JSON data to a schema-less database such as MongoDB.
+      </li>
+      <li><code>"filter"</code>: Only properties defined in the model are accepted.
+      If you load or save a model instance with properties that are not predefined, LoopBack will ignore them. This is particularly useful when dealing with old data that you wish to lose without a migration script.
+      </li>
+      </ul>
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>idInjection</code></td>
+    <td>Boolean</td>
+    <td><code>true</code></td>
+    <td>
+      Whether to automatically add an <code>id</code> property to the model:
+    <ul>
+    <li><code>true</code>: <code>id</code> property is added to the model automatically. This is the default.</li>
+    <li><code>false</code>: <code>id</code> property is not added to the model</li>
+    </ul>
+    See <a href="#id-properties">ID properties</a> for more information.  The <code>idInjection</code> property in <code>options</code> (if present) takes precedence.
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>name</code></td>
+    <td>String</td>
+    <td>None</td>
+    <td>Name of the model.</td>
+  </tr>
+
+  <tr>
+    <td><code>scopes</code></td>
+    <td>Object</td>
+    <td>N/A</td>
+    <td>See <a href="https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#scopes">Scopes</a> in lb3 docs.</td>
+  </tr>
+
+  </tbody>
+
+  </table>
+
+#### Unsupported Entries of Settings
+
+<table>
+  <thead>
+    <tr>
+      <th width="160">Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+  <tbody>
+
+  <tr>
+    <td><code>acls</code></td>
+    <td>
+      (TBD)
+    </td>
+  </tr>
+
+  <tr>
+  <td><code>base</code></td>
+  <td>(TBD))</td>
+  </tr>
+
+  <tr>
+  <td><code>excludeBaseProperties</code></td>
+  <td>(TBD))</td>
+  </tr>
+
+  <tr>
+  <td><code>http</code></td>
+  <td> In LB4 the http configurations are inferred from controller members and the rest server.</td>
+  </tr>
+
+  <tr>
+    <td><code>options</code></td>
+    <td>
+      (TBD)
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>plural</code></td>
+    <td>(TBD)</td>
+  </tr>
+
+  <tr>
+    <td><code>properties</code></td>
+    <td>This entry got removed from model decorator in lb4. See <code>@property decorator</code> to discover moer about how to define properties for your models.</td>
+  </tr>
+
+  <tr>
+    <td><code>relations</code></td>
+    <td>
+      In lb4, with the introduction of <a href="https://loopback.io/doc/en/lb4/Repositories.html">repositories</a>, now <code>relations</code> are defined by <code>relations decorators</code>.
+      See <a href="https://loopback.io/doc/en/lb4/Relations.html">Relations</a> for more details.
+    </td>
+  </tr>
+  <tr>
+    <td><code>remoting.<br/>normalizeHttpPath</code></td>
+    <td>
+    (TBD)
```

Comment:
This is again related to HTTP configuration.

I am not sure if we will ever support this field in LB4, the text "TBD" seems misleading to me.